.github/workflows/test.yml

```diff
@@ -63,16 +63,16 @@ jobs:
       strategy:
           matrix:
               php-versions: ['8.3']
-              nextcloud-versions: ['master']
+              nextcloud-versions: ['chore/mail-bisect-6e1d9a26209ec5524fbc2fb9c7cbb53315e64d72']
```

Comment:
This is because of https://github.com/nextcloud/mail/issues/10688#issuecomment-2662550235. I need a slightly outdated server checkout for CI. Right now that means we don't get the latest server master results, but I'm mostly interested in getting this to stable4.2